cmd/jaeger/internal/integration/badger_test.go

```diff
@@ -27,5 +27,5 @@ func TestBadgerStorage(t *testing.T) {
 	t.Cleanup(func() {
 		s.e2eCleanUp(t)
 	})
-	s.RunAll(t)
+	s.StorageIntegration.RunAll(t)
```

Comment:
why would this be related to OTEL upgrade?

Reply:
Yes, You are right. I will remove this unnecessary change.